package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "gulp-stylelint",
-  "version": "3.7.0",
+  "version": "3.7.1",
```

Comment:
There's no need to bump the lib version yourself (it should be a minor, not patch anyway). It will bumped before release.